services/simcore/docker-compose.yml

```diff
@@ -198,7 +232,13 @@ services:
       placement:
         constraints:
           - node.labels.simcore==true
-
+      resources:
+        reservations:
+          cpus: "0.1"
+          memory: "512M"
+        limits:
+          cpus: "2"
+          memory: "2G"
```

Comment:
same question here. how was this measured? I am also wondering if the webserver is capable to go above 1 CPU?